client/src/test/java/org/spine3/base/StringifiersShould.java

```diff
@@ -237,4 +238,14 @@ public void convert_event_id_to_string() {
 
         assertEquals(idToString(id), actual);
     }
+
+    @Test
+    public void pass_the_null_tolerance_check() {
+        final NullToleranceTest nullToleranceTest = NullToleranceTest.newBuilder()
+                                                                     .setClass(Stringifiers.class)
+                                                                     .addDefaultValue(CommandId.newBuilder())
```

Comment:
This default value does not look correct. As far as I can see, that's a value for an ID class? If so, please use some proper ID class value as a default.